package.json

```diff
@@ -1,41 +1,43 @@
 {
-    "name": "typescript",
-    "author": "Microsoft Corp.",
-    "homepage": "http://typescriptlang.org/",
-    "version": "1.5.2",
-    "license": "Apache-2.0",
-    "description": "TypeScript is a language for application scale JavaScript development",
-    "keywords": [
-        "TypeScript",
-        "Microsoft",
-        "compiler",
-        "language",
-        "javascript"
-    ],
-    "bugs": {
-        "url": "https://github.com/Microsoft/TypeScript/issues"
-    },
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/Microsoft/TypeScript.git"
-    },
-    "preferGlobal": true,
-    "main": "./bin/typescript.js",
-    "bin": {
-        "tsc": "./bin/tsc",
-        "tsserver": "./bin/tsserver"
-    },
-    "engines": {
-        "node": ">=0.8.0"
-    },
-    "devDependencies": {
-        "jake": "latest",
-        "mocha": "latest",
-        "chai": "latest",
-        "browserify": "latest",
-        "istanbul": "latest"
-    },
-    "scripts": {
-        "test": "jake runtests"
-    }
+  "name": "typescript",
```

Comment:
No, I noticed that too, will fix before I check in if this system works. Either Sublime or VSCode must've auto-reformatted me.